openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
> This is the concept on the whole redesign of fvSolution/fvSchemes/Time classes, since they were to designed to always read from disk and not in general disk/memory framework.

Right, I understand.  I am just suggesting a change to `TimeMod` (ovveride `run()` and remove `update()`) .  I can create a branch to illustrate what I mean if it wasn't clear in the 3 steps